<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>761</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Properties</string>
   </property>
   <widget class="QRadioButton" name="Positive_radio">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Positive Spectra</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Negative_radio">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Negative Spectra</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Element</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Min</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Max</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Molecular Mass</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>761</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Finder</string>
   </property>
   <widget class="QPushButton" name="find_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dalton_spinbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>67</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.000500000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="ppm_spinbox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>31</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="ppm_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ppm Difference</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dalton_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>68</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dalton Difference</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_SearchMethod">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>761</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Compounds list file</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="files_selectro_Layout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="EC_Open_btn">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EC_line"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="EC_label">
       <property name="text">
        <string>Elements file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
